tests/unit/test_retry.py

```diff
@@ -406,8 +407,17 @@ def increase_time(sleep_delay):
         last_wait = sleep.call_args.args[0]
         total_wait = sum(call_args.args[0] for call_args in sleep.call_args_list)
 
-        assert last_wait == 2.9  # and not 8.0, because the last delay was shortened
-        assert total_wait == 9.9  # the same as the deadline
+        assert last_wait == 8.0
```

Comment:
Why is it 8? i actually don't get

Reply:
Same thing as the other two - all are the effect of the removal of the rounding for the last attempt.